bigbluebutton-html5/imports/ui/components/media/webcam-draggable-overlay/component.jsx

```diff
@@ -313,6 +333,13 @@ export default class WebcamDraggableOverlay extends Component {
     }
   }
 
+  eventVideoFocusChangeListener() {
+    setTimeout(() => {
+      this.setInitialReferencePoint();
+      this.setLastWebcamPosition();
+    }, 500);
```

Comment:
Why do we need the timeout here?

Reply:
I need to wait for the component modification for recalculate the position.

Reply:
Can't the webcam pos/reference point calls be tethered to component state changes, then? I see that they're enveloped on async setTimeouts on almost every call when there's a change event of some sort (focus, resize). I worry it might be flaky on low end computers anyways (timeouts always are).

Reply:
Did it because this component need to look for the DOM. Can't use only the refs or state components. For this, use the react lifecycle causes a cascade events.